### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by Jonatino**

### CODE REVIEW

File: Comanche055/CONIC_SUBROUTINES.agc [hunk 1]
```diff
@@ -111,7 +111,7 @@
 #							MAJOR AXIS), FOR SPEED OF CONVERGENCE, BUT IF EITHER CONDITION FAILS, XKEPNEW IS RESET
 #							BY KEPLER TO A POOR BUT VALID GUESS.
 #	TC		+28			DP PREV. VALUE OF TIME IN CENTISECS. MUST BE LESS THAN ONE ORBITAL PERIOD.
-#	XPREV		+17 FOR EARTH		PRVIOUS VALUE OF X IN SQRT(METERS).  MUST BE LESS THAN AN X CORRESPONDING TO ONE
```
Comment (oldmud0) [inline Comanche055/CONIC_SUBROUTINES.agc]: Nope. Correct is DP PREV. http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1264.jpg

Comment (Jonatino) [inline Comanche055/CONIC_SUBROUTINES.agc]: @oldmud0 see 7b7f248

**Comment by wopian** [comment]
This was already part of my fixes in #149 

https://github.com/chrislgarry/Apollo-11/pull/149/files#diff-5ce9a96bed0ad3627bd56378a5401348R128

**Comment by oldmud0** [comment]
See I knew there was something wrong in me merging.

**Pull request merged**